### ISSUE TITLE
Golang new issue: filed per popular demand.

### ISSUE THREAD

**Issue by 62f** [ish-app/ish#1087]
@joefitzgerald That seems like an unrelated problem, could you open a new issue?

__Originally posted by @tbodt in https://github.com/ish-app/ish/issues/710#issuecomment-637256774__

go-1.13.15-r0 x86 {go} (BSD-3-Clause) [installed]
iPhone0 user root@Thu 26_11:41AM~(/root)-zsh:go --help
Go is a tool for managing Go source code.

Usage:

        go <command> [arguments]

The commands are:

        bug         start a bug report
        build       compile packages and dependencies
        clean       remove object files and cached files
        doc         show documentation for package or symbol
        env         print Go environment information
        fix         update packages to use new APIs
        fmt         gofmt (reformat) package sources
        generate    generate Go files by processing source
        get         add dependencies to current module and install them
        install     compile and install packages and dependencies
        list        list packages or modules
        mod         module maintenance
        run         compile and run Go program
        test        test packages
        tool        run specified go tool
        version     print Go version
        vet         report likely mistakes in packages

Use "go help <command>" for more information about a command.

Additional help topics:

        buildmode   build modes
        c           calling between Go and C
        cache       build and test caching
        environment environment variables
        filetype    file types
        go.mod      the go.mod file
        gopath      GOPATH environment variable
        gopath-get  legacy GOPATH go get
        goproxy     module proxy protocol
        importpath  import path syntax
        modules     modules, module versions, and more
        module-get  module-aware go get
        module-auth module authentication using go.sum
        module-private module configuration for non-public modules
        packages    package lists and patterns
        testflag    testing flags
        testfunc    testing functions

Use "go help <topic>" for more information about that topic.

iPhone0 user root@Thu 26_11:43AM~(/root)-zsh:go packages
go packages: unknown command
Run 'go help' for usage.

iPhone0 user root@Thu 26_11:43AM~(/root)-zsh:go list
can't load package: package .: no Go files in /root

iPhone0 user root@Thu 26_ 2:03PM~(/root)-zsh:go env
GO111MODULE=""
GOARCH="386"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOENV="/root/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="386"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/root/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_386"
GCCGO="gccgo"
GO386="387"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m32 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build732347496=/tmp/go-build -gno-record-gcc-switches"
iPhone0 user root@Thu 26_ 2:16PM~(/root)-zsh:go get -u github.com/ramya-rao-a/go-outline

#cd .; git clone -- https://go.googlesource.com/tools /root/go/src/golang.org/x/tools
waitid: interrupted system callpackage golang.org/x/tools/go/buildutil: waitid: interrupted system call
iPhone0 user root@Thu 26_ 2:26PM~(/root)-zsh:dmesg
[    0.000000] 15 stub syscall 375
               20 stub syscall 375
               25 stub syscall 375
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               pollnval 59 flags 4 events 3 revents 32
               30 stub syscall 375
               44 stub syscall 375
               49 stub syscall 375
               51 stub syscall 375
               pollnval 61 flags 4 events 3 revents 32
               pollnval 61 flags 4 events 3 revents 32
               pollnval 61 flags 4 events 3 revents 32
               pollnval 61 flags 4 events 3 revents 32
               pollnval 61 flags 4 events 3 revents 32
iPhone0 user root@Thu 26_ 2:27PM~(/root)-zsh:

**Comment by tbodt** [comment]
What's the bug here?

**Comment by 62f** [comment]
‘waitid: interrupted system call’

**Comment by lijhello** [comment]
Does `go build` work

**Comment by 62f** [comment]
> Does `go build` work?
> 
> 
> 
> I wanted to try Apple Swift on iSH the other day, but there's no apk package for it. I'll wait for Docker support on iSH and try Swift through Docker.

@lijhello no, ```go build``` has the same exact complaint about missing files.
I think Docker is 64-bit only.
